packages/web3-eth-accounts/src/common/common.ts

```diff
@@ -14,14 +14,14 @@ GNU Lesser General Public License for more details.
 You should have received a copy of the GNU Lesser General Public License
 along with web3.js.  If not, see <http://www.gnu.org/licenses/>.
 */
-import { buf as crc32Buffer } from 'crc-32';
+
 import { EventEmitter } from 'events';
 import type { Numbers } from 'web3-types';
 import { TypeOutput } from './types';
 import { intToBuffer, toType, parseGethGenesis } from './utils';
-import goerli from './chains/goerli.json';
-import mainnet from './chains/mainnet.json';
-import sepolia from './chains/sepolia.json';
+import goerli from './chains/goerli' ;
```

Comment:
If we use `Static import with JSON import assertion` it solves problem for ESM only, but as it creates another issue for CJS build outputs due to current configuration settings that we have, so I converted json to ts., supporting both ESM+CJS hybrid builds in our case plus users dnt need additional `--experimental-json-modules`.